src/main/java/com/bhavit/csvreadwrite/util/CsvHandler.java

```diff
@@ -12,10 +12,10 @@
 import java.io.IOException;
 
 /**
+ *
  * class that reads records from CSV file. and save to another CSV file.
  */
-public class CsvHandler {
-
+public class CsvHandler {    
```

Comment:
I'm sorry, but the provided information does not include any Java code to review. Please provide the Java code that you want reviewed so that I can assist you.